projects/plugins/boost/app/assets/src/js/pages/purchase-success/purchase-success.tsx

```diff
@@ -27,6 +28,7 @@ const PurchaseSuccess: React.FC = () => {
 	}, [] );
 
 	const wpcomPricingUrl = getRedirectUrl( 'wpcom-pricing' );
```

Comment:
This link was already here, but it seems wrong. It leads here: https://wordpress.com/pricing/

While we're updating this page, can we fix this link to lead to the Jetpack plans page instead of the wpcom one?

Reply:
Oh shoot! The link is right. I just switched the `isAtomic` check to see the paragraph.

I've reverted that in f8aa481.